CMakeLists.txt

```diff
@@ -78,7 +78,7 @@ gz_find_package(DART
     utils
     utils-urdf
   CONFIG
-  VERSION 6.9
+  VERSION 6.10
```

Comment:
please don't change the version here, restore this change

Reply:
dartsim version >= 6.10 is required, because it fixes the issues when [no ray hit](https://github.com/dartsim/dart/releases/tag/v6.10.0).

Do you have an alternative suggestion?

Reply:
I thought this was a minimal dependency only and, since the Harmonic targets Ubuntu 22.04 - which uses version 6.13 [by default](https://launchpad.net/~dartsim/+archive/ubuntu/ppa), this change wouldn't impact the main targeted OS. Although I'm not sure about the status of Harmonic's support for Windows, the corresponding CI did not complain about it.

Reply:
Can we resolve this thread? @azeey @ahcorde